extensions/parquet/table/src/main/java/io/deephaven/parquet/table/layout/ParquetFileHelper.java

```diff
@@ -1,12 +1,15 @@
 package io.deephaven.parquet.table.layout;
 
-import java.nio.file.Path;
+import io.deephaven.parquet.table.ParquetTableWriter;
 
-import static io.deephaven.parquet.table.ParquetTableWriter.PARQUET_FILE_EXTENSION;
+import java.nio.file.Path;
 
 final class ParquetFileHelper {
-
-    public static boolean fileNameMatches(Path path) {
-        return path.getFileName().toString().endsWith(PARQUET_FILE_EXTENSION);
+    /**
+     * Used as a filter to select relevant parquet files while reading all files in a directory
+     */
+    static boolean parquetFileFilter(Path path) {
+        return path.getFileName().toString().endsWith(ParquetTableWriter.PARQUET_FILE_EXTENSION) &&
+                !path.toFile().isHidden();
```

Comment:
I'm not sure this is correct on all OS (i.e. Windows is stupid).  We are treating the dot prefix as our marker for the folder names, should we just test for that instead of relying on OS-dependent file-hidden tests?

Or is this something totally different?  I may be missing the point.